source/includes/common/installation.rst

```diff
@@ -10,6 +10,14 @@ Install and Deploy MinIO
    :local:
    :depth: 1
 
+.. container:: extlinks-video
```

Comment:
@djwfyi this ended up working well enough, and it's dirt simple. We could change the classname if we wanted to.

Admonitions come with a lot of other classes attached, which made it a little harder to script / style without accidentally munging something else.